Errors due to ```make html```

## Expected behavior ##
- When ```make html``` command executed in side ```mdanalysis/package/doc/sphinx``` directory then it must create html for the docs
<!-- A clear and concise description of what you want to do and what you think should happen. (Code to reproduce the behavior can be added below). -->


## Actual behavior ##
#### Currently it gives two errors:
1) Due to ```html_static_path```
```bash
Warning, treated as error:
html_static_path entry '_static' does not exist
make: *** [Makefile:42: html] Error 2
```

2) Due to indentation:
```bash
sphinx.errors.SphinxWarning: /home/heet/Work/Development/mdanalysis/package/MDAnalysis/coordinates/FHIAIMS.py:docstring of MDAnalysis.coordinates.base.ProtoReader.timeseries:8:Explicit markup ends without a blank line; unexpected unindent.
```
<!-- What happened instead. Add as much detail as you can. Include (copy and paste) stack traces and any output. -->


## Code to reproduce the behavior ##

<!-- Show us how to reproduce the failure. If you can, use trajectory files from the test data. Use the code snipped below as a starting point. -->

``` bash
cd mdanalysis/package/doc/sphinx && make html

```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
  - 2.7.0-dev0
- Which version of Python (`python -V`)?
  - Python 3.10.13
- Which operating system?
  - Ubuntu

@lilyminium

- After doing `html_static_path = []` it still gives error like 
```bash

Warning, treated as error:
/home/heet/Work/Development/mdanalysis/package/MDAnalysis/coordinates/FHIAIMS.py:docstring of MDAnalysis.coordinates.base.ProtoReader.timeseries:8:Explicit markup ends without a blank line; unexpected unindent.
make: *** [Makefile:42: html] Error 2
```

should be fixed in the same PR ?
- If yes then please guide me how can i tackle it.

--- a/package/CHANGELOG
+++ b/package/CHANGELOG
@@ -20,6 +20,7 @@ The rules for this file:
  * 2.7.0
 
 Fixes
+  * Fix `make html` command errors (Issue #4362, PR #4365)
   * Updated cimport for numpy to maintain cimport consistency (Issue #3908)
   * Fix `NoJump` unwrapping for jumps on the second frame in a
     trajectory (PR #4258, Issue #4257)

--- a/package/CHANGELOG
+++ b/package/CHANGELOG
@@ -20,7 +20,8 @@ The rules for this file:
  * 2.7.0
 
 Fixes
-  * Fix `make html` command errors (Issue #4362, PR #4365)
+  * Fix documentation building errors due to `html_static_path` and unindented
+    block of code (Issue #4362, PR #4365)
   * Updated cimport for numpy to maintain cimport consistency (Issue #3908)
   * Fix `NoJump` unwrapping for jumps on the second frame in a
     trajectory (PR #4258, Issue #4257)

--- a/package/MDAnalysis/coordinates/base.py
+++ b/package/MDAnalysis/coordinates/base.py
@@ -1001,7 +1001,7 @@ class ProtoReader(IOBase, metaclass=_Readermeta):
             of coordinate data is returned.
 
             .. deprecated:: 2.7.0
-            asel argument will be renamed to atomgroup in 3.0.0
+                asel argument will be renamed to atomgroup in 3.0.0
 
         atomgroup: AtomGroup (optional)
             Same as `asel`, will replace `asel` in 3.0.0

--- a/package/doc/sphinx/source/conf.py
+++ b/package/doc/sphinx/source/conf.py
@@ -190,7 +190,7 @@ html_context = {
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 # For RTD theme: custom.css to override theme defaults.
-html_static_path = ['_static']
+html_static_path = []
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.

